package.json

```diff
@@ -105,4 +115,4 @@
   "prettier": {
     "singleQuote": true
   }
```

Comment:
I know that we have https://github.com/Vivify-Ideas/react-native-boilerplate/blob/master/.prettierrc.js so not sure what's this required. 
Probably we should delete this since there are different ways prettier rules can be defined.
Also, make sure as we talked that the same react-boilerplate prettier rules (without semicolons) are applied here too.